spec.md

```diff
@@ -0,0 +1,728 @@
+Those that can query about other accounts are prefixed with `getExternalAccount`.
+
+#### `EEI.getAccountBalance`
+
+Return the balance of the current account (`ACCT`).
+
+1.  Load the value `ACCT` from `eei.call.id`.
+
+2.  Load and return the value `eei.accounts[ACCT].balance`.
+
+```k
+    syntax EEIMethod ::= "EEI.getAccountBalance"
+ // --------------------------------------------
+    rule <k> EEI.getAccountBalance => BAL ... </k>
+         <id> ACCT </id>
+         <account>
+           <acctID>  ACCT </acctID>
+           <balance> BAL  </balance>
+           ...
+         </account>
+```
+
+#### `EEI.getAccountCode`
+
+Get the code of the given account `ACCT`.
+
+1.  Load the value `ACCT` from `eei.call.id`.
+
+2.  Load and return `eei.accounts[ACCT].code`.
+
+```k
+    syntax EEIMethod ::= "EEI.getAccountCode"
+ // -----------------------------------------
+    rule <k> EEI.getAccountCode => ACCTCODE ... </k>
+         <id> ACCT </id>
+         <accounts>
+           <acctID> ACCT </acctID>
+           <code> ACCTCODE </code>
+           ...
+         </accounts>
+```
+
+#### `EEI.getExternalAccountCode : Int`
+
+Get the code of the given account `ACCT`.
+
+1.  Load and return `eei.accounts[ACCT].code`.
+
+```k
+    syntax EEIMethod ::= "EEI.getExternalAccountCode" Int
+ // -----------------------------------------------------
+    rule <k> EEI.getExternalAccountCode ACCT => ACCTCODE ... </k>
+         <accounts>
+           <acctID> ACCT </acctID>
+           <code> ACCTCODE </code>
+           ...
+         </accounts>
+```
+
+#### `EEI.getAccountStorage : Int`
+
+Returns the value at the given `INDEX` in the current executing accounts storage.
+
+1.  Load `ACCT` from `eei.call.id`.
+
+2.  If `eei.accounts[ACCT].storage[INDEX]` exists:
+
+    i.  then: return `eei.accounts[ACCT].storage[INDEX]`.
+
+    ii. else: return `0`.
+
+```k
+    syntax EEIMethod ::= "EEI.getAccountStorage" Int
+ // ------------------------------------------
+    rule <k> EEI.getAccountStorage INDEX => VALUE ... </k>
+         <id> ACCT </id>
+         <account>
+           <acctID> ACCT </acctID>
+           <storage> ... INDEX |-> VALUE ... </storage>
+           ...
+         </account>
+
+    rule <k> EEI.getAccountStorage INDEX => 0 ... </k>
+         <id> ACCT </id>
+         <account>
+           <acctID> ACCT </acctID>
+           <storage> STORAGE </storage>
+           ...
+         </account>
+      requires notBool INDEX in_keys(STORAGE)
+```
+
+#### `EEI.setAccountStorage : Int Int`
+
+At the given `INDEX` in the executing accounts storage, stores the given `VALUE`.
+
+1.  Load `ACCT` from `eei.call.id`.
+
+2.  Set `eei.accounts[ACCT].storage[INDEX]` to `VALUE`.
+
+```k
+    syntax EEIMethod ::= "EEI.setAccountStorage" Int Int
+ // ----------------------------------------------------
+    rule <k>  EEI.setAccountStorage INDEX VALUE => . ... </k>
```

Comment:
going be **that** guy and point out that there are two spaces between `<k>` and `EEI...`